sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/package-info.java

```diff
@@ -2,6 +2,121 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing clients for querying logs and metrics from Azure Monitor.
+ * <p>Azure Monitor Query service is a powerful tool that allows you to query and analyze log data from various sources
+ * in Azure. It is built on top of the Kusto Query Language (KQL), which is a powerful query language that allows you
```

Comment:
Only logs query uses Kusto Query Language. Metrics queries don't. So, we should update this doc to differentiate.